README.md

```diff
@@ -86,59 +86,57 @@ Known good configuration for EC2 instance:
        "deployDataIntake": <false || true deploy data intake>
    }
    ```
-
-4. Create `target_auth.json` under `guidance-for-processing-overhead-imagery-on-aws/lib/accounts/` and leave it blank unless you are planning to enable Authenication. If so, head over to [Enabling Authentication](#enabling-authentication) in this README.
-
-5. Export your dev account number and deployment username:
+   
```

Comment:
I would suggest leaving this in here. Probably rename to:

4. Optional: If you want to enable Authentication, please head over to [Enabling Authentication](#enabling-authentication) in this README.

Reply:
done